src/componenets/AllProducts.js

```diff
@@ -0,0 +1,67 @@
+import React, { useState } from "react";
+import "./AllProducts.css";
+import Product from "./Product";
+
+function getAllProducts(productsList, n, sortBy) {
+  let sortedProducts = [...productsList];
+
+  if (sortBy === "latest") {
+    sortedProducts.sort((a, b) => new Date(b.updatedAt) - new Date(a.updatedAt));
+  } else if (sortBy === "favorites") {
```

Comment:
이런 경우는 elif로 조건문이 체이닝되는것보다는 조건문에 맞는 결과를 명확하게 보여주는게 더 흐름을 파악하기 수월할것같아요 :)

e.g
```suggestion
  return;
  }; 
  
  if (sortBy === "favorites") {
  ...
  return;
  }
```